build.sbt

```diff
@@ -28,13 +28,13 @@ val coreDependencies = Seq(
 )
 
 val commonSettings =
-sonatypeSettings ++ scalariformSettings ++ Seq(
+  scalariformSettings ++ Seq(
   version := "0.11.0-SNAPSHOT",
-  organization := "com.softwaremill.reactivekafka",
+  organization := "com.typesafe.akka",
+  organizationName := "Lightbend",
```

Comment:
There's organisation in the maven metadata: `       <connection>scm:git:git@github.com:softwaremill/reactive-kafka.git</connection>`